data-publisher/src/main/kotlin/no/nav/amt/tiltak/data_publisher/model/EndringsmeldingPublishDto.kt

```diff
@@ -0,0 +1,20 @@
+package no.nav.amt.tiltak.data_publisher.model
+
+import no.nav.common.json.JsonUtils
+import org.springframework.util.DigestUtils
+import java.time.LocalDateTime
+import java.util.*
+
+data class EndringsmeldingPublishDto(
+	val id: UUID,
+	val deltakerId: UUID,
+	val utfortAvNavAnsattId: UUID?,
+	val opprettetAvArrangorAnsattId: UUID,
+	val utfortTidspunkt: LocalDateTime?,
+	val status: String,
+	val type: String,
+	val innhold: String,
```

Comment:
Bør vi ikke bruke objekt for innholdet? Hvis den eksisterende strukturen krangler med jackson så kan du gjøre sånn som dette: https://github.com/navikt/amt-tiltaksarrangor-bff/blob/main/src/main/kotlin/no/nav/tiltaksarrangor/controller/request/EndringsmeldingRequest.kt